### PR TITLE
feat(scripts): use pg instead of supabase

### PR DESCRIPTION
**Pull request by fiji-flo**
## Summary

Move to node-postgres to enable local postgres without supabase and enable us to move to our central DB in the future.

## How did you test this change?

Locally

### CODE REVIEW

File: scripts/ai-help-macros.ts [hunk 1]
```diff
@@ -56,7 +57,14 @@ export async function updateEmbeddings(
   }
 
   // Supabase.
```
Comment (argl) [inline scripts/ai-help-macros.ts:59]: Super tiny little nitty-nit: At this point, it is not about Supabase any more, it should be Postgres.

File: scripts/ai-help-macros.ts [hunk 2]
```diff
@@ -72,14 +80,11 @@ export async function updateEmbeddings(
       });
     } catch (e: any) {
       const {
-        data: {
-          error: { message, type },
-        },
+        error: { message, type },
         status,
-        statusText,
       } = e.response;
```
Comment (argl) [inline scripts/ai-help-macros.ts:85]: I had to change `e.response` to `e` here, otherwise the error could not be destructured properly.

**Review by argl — APPROVED** [review]
lgtm

**Review by caugner — APPROVED** [review]
Stage build LGTM: https://github.com/mdn/yari/actions/runs/7696995653/job/20973023914#step:27:31

**Pull request merged**